lib/src/localizations.dart

```diff
@@ -156,6 +156,13 @@ class InsporationLocalizations {
     locale: localeName
   );
 
+  String get errorSignInNoHost => Intl.message(
+      'Unable to resolve host, are you sure this pod supports the API?',
```

Comment:
The message seems misleading. Not being able to resolve the hostname tells us nothing about whether the pod supports the API or not.

Reply:
Good point. 
Should we insert an extra layer before auth process to check if service supports the api?
Maybe after "detect configuration" and in case of a error?

Reply:
How would you detect it?

Reply:
🤔
What do you think to pin this on 'Discover service'? On new or unknown services, this is the first call. If we receive no answer, server is unavailable or doesn't know this service at all (invalidate API)

Reply:
But what error conditions would  lead to this invalidation?

The one you picked here (while burying many others under this message) is not indicative of the API being available at the pod or not at all.